LSPS1/README.md

```diff
@@ -0,0 +1,409 @@
+* `<node> '@' <tor_onion_v3_address> ':' <port>`, with the last `:` after `@` considered a separator for the port number.
+
+Word definition:
+
+* `<node>` MUST be a node id (pubkey). For example: "0200000000a3eff613189ca6c4070c89206ad658e286751eca1f29262948247a5f".
+* `<ip>` MUST be a IPv4 OR IPv6 address.
+* `<tor_onion_v3_address>` MUST be a tor onion v3 address.
+* `<port>` MUST be a [port](https://en.wikipedia.org/wiki/Port_(computer_networking)) number.
+
+### Onchain addresses
+
+- All onchain addresses MUST be a bech32 version 0 ("SegWit") or bech32m version 1 ("Taproot") address or null.
+- The lsp/user MAY support other address types.
+
+### Actors
+
+`LSP` is the API provider. `User` is the user/client of the API.
+
+### Channel sides
+
+`user_balance` are the funds on the user side. `lsp_balance` are the funds on the LSP side.
+
+### Overprovisioning
+
+The LSP is allowed to overprovision channels/onchain-payments/onchain-fees as long as it benefits the user.
+
+> **Rationale** The lsp may need to "bin" UTXOs.
+
+## Flow
+
+### API information
+
+`GET /lsp/channels` is the entrypoint for each client using the api. It lists the version of the api and all supported extensions in a dictionary.
+
+The user MUST pull `GET /lsp/channels` to read
+
+- the `version` of the api AND version of `options` and therefore prove compatibility.
+- `options` properties to determine the api boundries.
+
+Example `GET /lsp/channels` response: 
+
+```JSON
+{
+  "version": 2,
+  "website": "http://example.com/contact",
+  "options": {
+    "base_api": {
+      "version": 1,
+      "max_user_balance_satoshi": "0",
+      "max_lsp_balance_satoshi": "100000000",
+      "min_required_onchain_satoshi": null,
+      "max_channel_expiry_weeks": 24
+    }
+  }
+}
+```
+
+#### Base api options
+
+The base api itself has multiple properties that MUST be defined.
+
+```json
+"options": {
+    "base_api": {
+        "version": 1,
+        "max_user_balance_satoshi": "0",
+        "max_lsp_balance_satoshi": "100000000",
+        "min_required_onchain_satoshi": null,
+        "max_channel_expiry_weeks": 24
+    }
+}
+```
+
+- `version` MUST be 1.
+- `max_user_balance_satoshi` MUST be the maximum number of satoshi that the LSP is willing to push to the user. MUST be 0 or a positive integer.
+- `max_lsp_balance_satoshi` MUST be the maximum number of satoshi that the LSP is willing to contribute to the their balance.  MUST be 1 or greater.
+- `min_required_onchain_satoshi` MUST be the number of satoshi (`order_total_satoshi` see below) that are required for the user to pay funds onchain. The LSP MUST allow onchain payments equal or above this value. MAY be null if onchain payments are NOT supported.
+- `max_channel_expiry_weeks` MUST be the maximum length in weeks a channel can be leased for. MUST be 1 or greater.
+
+The user MAY abort the flow here.
+
+### 2. Create Order
```

Comment:
You started numbering this with `2.` but the previous subsection "API Information" was not numbered.